R/theme_gtsummary_roche.R

```diff
@@ -28,23 +30,54 @@ theme_gtsummary_roche <- function(set_theme = TRUE, font_size = NULL) {
       list(
         "tbl_summary-fn:percent_fun" = gtsummary::label_style_number(digits = 1, scale = 100),
         "pkgwide-fn:pvalue_fun" = label_roche_pvalue(),
+        "pkgwide-fun:pre_conversion" = gtsummary::bold_labels(),
         "pkgwide-str:print_engine" = "flextable"
       )
     )
 
+
   # use arial font for flextables ----------------------------------------------
   lst_theme$`as_flex_table-lst:addl_cmds` <-
     c(
       lst_theme$`as_flex_table-lst:addl_cmds`,
-      list(valign = rlang::expr(flextable::font(fontname = "Arial", part = "all")))
+      list(
+        fontsize = rlang::expr(flextable::fontsize(size = !!font_size - 1, part = "footer")),
+        border = rlang::expr(
+          flextable::border_outer(part = "body", border = !!border)
+        ),
+        border = rlang::expr(
+          flextable::border_outer(part = "header", border = !!border)
+        ),
+        valign = rlang::expr(flextable::valign(valign = "top", part = "all")),
+        valign = rlang::expr(flextable::font(fontname = "Arial", part = "all")), # valign only bc last cmd
+        valign = rlang::expr(flextable::line_spacing(space = 1, part = "all"))
+      )
     )
+  # todo:
+  # * vertical padding different for label rows (3 + 0.2)
```

Comment:
I would not keep this as a feature. It seems unnecessary to me (it was in the requirements, though)